services-js/commissions-app/src/client/common/CommentInput.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export interface Props {
+  name: string;
+  placeholder: string;
+  title: string;
+  error?: string;
+  value: string;
+  onChange: any;
+  onBlur: any;
+}
+
+export default class Comments extends React.Component<Props> {
+  render() {
+    return (
+      <div className="txt">
+        <label className="txt-l" />
```

Comment:
Do you want `{this.props.title}` in here?

Reply:
Oh, I can see you're not really using it for this element. In that case, I'd recommend either:

 - Make `title` an optional prop and don't show `<label>` at all when it’s not there.
 - Remove `title` and `<label>` altogether from this component.

As it is the `title` prop is just ignored by this component, which is bound to confuse someone down the road.

Reply:
I made it an option, Well I made it optional the best way I know how